lib/public/components/Filters/common/filtersPanelPopover.js

```diff
@@ -14,6 +14,22 @@ import { popover } from '../../common/popover/popover.js';
 import { PopoverTriggerPreConfiguration } from '../../common/popover/popoverPreConfigurations.js';
 import { PopoverAnchors } from '../../common/popover/PopoverEngine.js';
 import { h } from '/js/src/index.js';
+import { profiles } from '../../common/table/profiles.js';
+import { applyProfile } from '../../../utilities/applyProfile.js';
+
+/**
+ * Adapt column profiles for applyProfile function
+ *
+ * @param {string|string[]|object<string, *>} profilesToBeAdapted profiles
+ * @return {string[]|object<string, *>} adapted profile
+ */
+const adaptColumnProfiles = (profilesToBeAdapted) => {
```

Comment:
Please, make an effort on naming and conventions!!!!
Adapt to what? What does it looks like, to what do you adapt it?
If a function make sense in a given context only (`applyProfile` function) then it must be inside of it, you must not tell in the documentation where the function must be used, it goes completely against the reusability common sense

Reply:
removed